### PR TITLE
fix(odata-service-inquirer): CAP loading errors not displayed

### PR DESCRIPTION
**Pull request by IainSAP**
https://github.com/SAP/open-ux-tools/issues/2604

Fixes incorrectly displayed CAP loading errors.

image

image

### CODE REVIEW

**Review by cianmSAP — APPROVED** [review]
Updates look good
Not tested locally
Changeset present & tests to cover

File: packages/inquirer-common/src/translations/inquirer-common.i18n.json [hunk 1]
```diff
@@ -14,9 +14,6 @@
             "missingPropMsg": "The property: `{{destinationProperty}}` is missing.",
             "httpConnectionError": "A connection error occurred with the selected destination. Http code: {{- code}}. $t(texts.seeLogForDetails)"
         },
-        "cannotReadCapServiceMetadata": "An error occurred reading CAP service metadata: {{serviceName}}. $t(texts.seeLogForDetails).",
-        "capModelAndServicesLoadError": "An error occurred loading the CAP model and services. {{- error}}",
-        "capServiceUrlPathNotDefined": "An error occurred reading CAP service metadata: {{serviceName}}. CAP service property `urlPath` is not defined but is required.",
         "unknownError": "An error occurred{{- error, addMsgWithColonFormatter}}",
```
Comment (longieirl) [inline packages/inquirer-common/src/translations/inquirer-common.i18n.json:17]: Do you need a space after `...occurrred`?

Comment (IainSAP) [reply packages/inquirer-common/src/translations/inquirer-common.i18n.json:17]: Its handled by the formatter: https://github.com/SAP/open-ux-tools/blob/main/packages/odata-service-inquirer/src/i18n.ts#L24

**Review by longieirl — APPROVED** [review]
Text changes look good.
Issues addressed.

**Pull request merged**